apps/docs/pages/guides/platform/database-usage.mdx

```diff
@@ -37,6 +42,13 @@ If you need more than 1024TB of database storage or require multiple storage exp
 
    This allows you to delete data from within the session.
 
+### Preoccupied Space
```

Comment:
Is there anything besides extensions that affect db size with a new project? If it's just extensions, might make sense to rename this section to something specific to extensions or something like "Usable database size" otherwise.

Reply:
Maybe @pcnc can elaborate this a bit better, what exactly is using up database space when launching a new database? Just extensions or other stuff, too?

Reply:
Extensions, schema/data used by our services that are offered with each project, default PG things like template0/template1 db
https://www.dbi-services.com/blog/what-the-hell-are-these-template0-and-template1-databases-in-postgresql/

Reply:
Adjusted the wording